.buildkite/scripts/jdk-matrix-tests/generate-steps.py

```diff
@@ -81,21 +83,24 @@ def __init__(self, os: str, jdk: str, group_key: str):
 
     def all_jobs(self) -> list[typing.Callable[[], JobRetValues]]:
         return [
+          self.init_annotation,
           self.unit_tests,
         ]
 
     def unit_tests(self) -> JobRetValues:
-        step_name_human = "Java Unit Test"
-        test_command = "# TODO"
+        step_name_human = "Unit Test"
```

Comment:
The only neat here is to maybe do "Unit Tests (Java/Ruby)"

Reply:
Great idea. Added it in https://github.com/elastic/logstash/pull/15563/commits/491fed51ecb017b7b47460045f3078f393abcb02

Reply:
raised https://github.com/elastic/logstash/issues/15566 to track the improvement